ngraph/frontend/tensorflow/src/ngraph_conversions.hpp

```diff
@@ -67,7 +65,7 @@ void NHWCtoHW(const std::vector<T>& src, std::vector<size_t>& dst) {
 }
```

Comment:
very strange function and seems useless functions. Is it used anywhere? Because permutation order can be formed inside `insert_transpose` function. If yes, propose to rename it. Or better to remove it

Reply:
need to be analyzed in more detail
let's do it in the next PR separately